www/md_base/src/icons/collapse-panel.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
These two SVGs are drawing by myself because I did't find other icon set
that provides the icons I think express the function well.

This might be an advantage of using SVG :)

On Sun, May 17, 2015 at 4:37 PM Pierre Tardy <notifications@github.com>
wrote:

> In www/md_base/src/icons/collapse-panel.svg
> <https://github.com/buildbot/buildbot/pull/1665#discussion_r30468545>:
>
> > @@ -0,0 +1,13 @@
> > +<?xml version="1.0" encoding="UTF-8" standalone="no"?>
>
> more svgs :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/1665/files#r30468545>.
>